core/process.py

```diff
@@ -75,6 +75,42 @@
 STYLE_PID_ONLY = 2
 
 
+class ctx_IgnoreTtySignals(object):
```

Comment:
only used in `GiveTerminal()` and might not be strictly necessary, but is good hygiene?

Reply:
ended up being unnecessary